lib/python/picongpu/extra/input/bunchInit_openPMD_bp.py

```diff
@@ -422,7 +420,7 @@ def __init__(
         outfile,
         particles=[],
         inconfig="{}",
-        outconfig="{}",
+        outconfig="adios2.engine.parameters.BufferChunkSize = 2147381248",
```

Comment:
Where does this number come from?
Please add a comment on the number.

Reply:
@franzpoeschel added this parameter when overlooking the script.
In the [ADIOS2 Documentation](https://adios2.readthedocs.io/en/latest/engines/engines.html#bp5) it is stated that it can be used to increase write performance:

> BufferChunkSize: (for chunk buffer type) The size of each memory buffer chunk, default is 128MB but it is worth increasing up to 2147381248 (a bit less than 2GB) if possible for maximum write performance.

I didn't notice the change but I can add a comment and check if it makes a difference in performance.

Reply:
I think there's no need to check the performance as in this script it likely makes no difference. Given the data sizes that we're dealing with here, there's just no use in *not* setting this parameter to the maximum possible value, especially since on Linux systems there is basically no downside due to how those systems handle virtual memory. Linking the ADIOS2 documentation here should be enough.

Reply:
I can verify that it makes no difference to the performance but I will keep it and add a comment.

Reply:
I mostly expect that it could make a difference if we switch to writing smaller block sizes